src/components/DappBrowser/search-input/AccountIcon.tsx

```diff
@@ -23,8 +23,8 @@ export const AccountIcon = React.memo(function AccountIcon() {
   const [currentAddress, setCurrentAddress] = useState<string>(accountAddress);
 
   const { activeTabRef } = useBrowserContext();
-  const activeTabHost = useBrowserStore(state => getDappHost(state.getActiveTabUrl())) || DEFAULT_TAB_URL;
-  const isOnHomepage = useBrowserStore(state => (state.getActiveTabUrl() || DEFAULT_TAB_URL) === RAINBOW_HOME);
+  const activeTabHost = useBrowserStore(state => getDappHost(state.getActiveTabUrl())) || RAINBOW_HOME;
+  const isOnHomepage = useBrowserStore(state => (state.getActiveTabUrl() || RAINBOW_HOME) === RAINBOW_HOME);
```

Comment:
was just curious what was the motivation for removing the usage of `DEFAULT_TAB_URL`  with `RAINBOW_HOME` everywhere?

Reply:
it was just a re-export of the same string as RAINBOW_HOME which was unnecessary.